### PR TITLE
[DELIVERABLE 1] Add Section 2 + 8 (incomplete)

### PR DESCRIPTION
**Pull request by tbrar06**

### CODE REVIEW

**Review by Sevhena — APPROVED** [review]
Very nice!

**Comment by Sevhena** [comment]
I've added, the assignee, label, milestone, project and issue but please try to remember to do so in the future thanks

File: .gitignore [hunk 1]
```diff
@@ -277,3 +277,6 @@ TSWLatexianTemp*
 
 # DS_Store file from Mac computers
 .DS_Store
+
+#IDEA project settings and configuration files
+.idea/
```
Comment (nivethakuruparan) [inline .gitignore:282]: What is this doing?

**Review by nivethakuruparan — APPROVED** [review]
Other than the comment above, everything else LGTM

File: .gitignore (hunk 1 above)
Comment (tbrar06) [reply .gitignore:282]: For anyone using Intellij it makes sure IDE specific project setting changes are not auto committed

**Review by mmyaaaaa — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (nivethakuruparan) [reply .gitignore:282]: Wait are you using Intellij?

Comment (nivethakuruparan) [reply .gitignore:282]: But you can merge this for now since it doesn't affect anything

**Review by Sevhena — APPROVED** [review]

**Pull request merged**